app/services/paypal_service/api/payments.rb

```diff
@@ -37,7 +36,7 @@ def request(community_id, create_payment, force_sync: true)
       @lookup.with_active_account(
         community_id, create_payment[:merchant_id]
       ) do |m_acc|
-        if (use_async?(force_sync))
+        if !force_sync
```

Comment:
Is the idea to remove the async path for these from paypal service altogether now that the transaction service level handles these operations asynchronously?